### DISCUSSION TITLE
如果是自己申请的ORC接口,怎么在MouseInc里边对接呢?

### DISCUSSION THREAD

**Discussion by SoyKoa** [shuax/MouseInc.Settings#432] [Q&A]
![image](https://user-images.githubusercontent.com/76244289/191895108-7ab39025-676b-41ad-b374-76f13d0e6f15.png)
老大,因为经常要用ORC功能,,,有道速度快,但经常次数限制而报错,,,飞桨速度太慢,,我就自己去百度申请了个key,个人用户每月也足够用了,,但在orc设置页面,没看到怎么把key加进去呢?

**Comment by shuax** [comment]
还不支持自定义
